benchmarks/Neo.VM.Benchmarks/OpCode/BenchmarkEngine.cs

```diff
@@ -0,0 +1,190 @@
+// Copyright (C) 2015-2024 The Neo Project.
+//
+// BenchmarkEngine.cs file belongs to the neo project and is free
+// software distributed under the MIT software license, see the
+// accompanying file LICENSE in the main directory of the
+// repository or http://www.opensource.org/licenses/mit-license.php
+// for more details.
+//
+// Redistribution and use in source and binary forms with or without
+// modifications are permitted.
+
+using Neo.Test.Types;
+using System.Diagnostics;
+using System.Runtime.CompilerServices;
+
+namespace Neo.VM.Benchmark.OpCode;
```

Comment:
some files has block namespace, and others don't

Reply:
updated.